lib/cn/kvset.c

```diff
@@ -749,11 +749,11 @@ kvset_open2(
             if (!km->km_restored) {
                 kvset_madvise_hblk(ks, MADV_WILLNEED, true);
 
-                if (ra_willneed(rp->cn_mcache_kra_params)) {
+                if (ra_willneed(rp->cn_mmap_kra_params)) {
                     kvset_madvise_kblks(ks, MADV_WILLNEED, true, true);
                 }
 
-                if (ra_willneed(rp->cn_mcache_vra_params)) {
+                if (ra_willneed(rp->cn_mmap_vra_params)) {
```

Comment:
`ra_willneed` is defined as:
```
#define ra_willneed(_ra)    ((_ra) & 0x01u)
```
Seems we can change param name to `cn_mmap_vra` and eliminate `ra_willneed`.

Reply:
It used to contain a lot of stuff (for level based cached), but no longer, so yeah, change the name.
We should probably ignore these settings if the kblocks/vblocks are larger than RAM, nothing good can happen.  I could see maybe enabling them on a small, hot kvs, but used indiscriminantly can cause a world of hurt.